admin/autoupdater/src/autoupdater.c

```diff
@@ -334,6 +343,17 @@ static bool autoupdate(const char *mirror, struct settings *s, int lock_fd) {
 		goto out;
 	}
 
+	if (s->check) {
+		printf(
+			"VERSION=%s\n"
+			"SIZE=%u\n",
```

Comment:
imagesize is defined as an ssize_t, so the correct conversion pattern would be `%zd`.